typhos/cli.py

```diff
@@ -96,7 +114,15 @@ def _create_happi_client(cfg):
 
 
 def create_suite(devices, cfg=None, fake_devices=False):
-    """Create a TyphosSuite from a list of device names"""
+    """Create a TyphosSuite from a list of device names."""
+    if devices:
```

Comment:
Fails when using `typhos` with no devices (unbound loaded_devs on line 120)

Reply:
Oops! Guess I broke it.